packages/app/src/server/routes/api/cortex/cortexRuntimeConfig.ts

```diff
@@ -105,6 +104,14 @@ export default async function setCortexRuntimeConfigHandler(
   res: Response,
   next: NextFunction
 ) {
+  if (!req.is("text/plain")) {
+    return next(
+      new GeneralServerError(
```

Comment:
@jgehrcke by using this error handling technique we can present a consistent error handling mechanism to the client, like this: https://github.com/opstrace/opstrace/pull/800/commits/fa9959acaa1f72957630c19744afd2573b288781#r642102300